plugins/test/unit/plugins/importers/yum/test_sync.py

```diff
@@ -1,7 +1,11 @@
 from copy import deepcopy
 from cStringIO import StringIO
 import os
-import unittest
+
+try:
+    import unittest2 as unittest
+except ImportError:
```

Comment:
EL6 will require unittest2. My related platform PR addresses that in the spec file and in the puppet module that our test automation uses.